yt/utilities/parallel_tools/task_queue.py

```diff
@@ -176,7 +176,7 @@ def dynamic_parallel_objects(tasks, njobs=0, storage=None, broadcast=True):
             for task in my_q:
                 rstore = ResultsStorage()
                 yield rstore, task
-                my_q.send_result(rstore.result)
+                my_q.send_result(rstore)
```

Comment:
should the def of `send_result` be updated to  `def send_result(self, rstore):` too? or is rstore not always passed to it?

Reply:
Sure, I can do that.